lib/PasswordValidationField/tests/PasswordValidationField-test.js

```diff
@@ -47,83 +45,79 @@ describe('PasswordValidationField', () => {
         totalRecords: 1
       });
     });
-
-    // mount the component under test
-    mount(
-      <TestForm>
-        <ConnectedField
-          id="password-test"
-          name="password"
-          label="Test"
-          username="test"
-        />
-      </TestForm>
-    );
-
-    // wait for validation to load
-    await when(() => rulesLoaded);
-  });
-
-  it('is a textfield component', () => {
-    expect(field).to.not.be.undefined;
-  });
-
-  describe('with an invalid password', () => {
-    beforeEach(async () => {
-      await field.focus();
-      await field.fillIn('test');
-      await field.blur();
-    });
-
-    it('shows a validation error for invalid length', () => {
-      expect(field.has({ error: 'password.length.invalid' }));
-    });
   });
 
-  describe('with valid password and final form', () => {
+  describe('using redux-form', () => {
     beforeEach(async () => {
-      mount(
+      // mount the component under test
+      await mount(
         <TestForm>
           <ConnectedField
             id="password-test"
             name="password"
             label="Test"
             username="test"
-            fieldClass="final-form"
           />
         </TestForm>
       );
-      await field.focus();
-      await field.fillIn('test-string');
-      await field.blur();
+
+      // wait for validation to load
+      await converge(() => { if (!rulesLoaded) throw new Error('rules not loaded'); });
     });
 
-    it('doesnt show a validation error', () => {
-      expect(field.has({ error: '' }));
+    describe('with an invalid password', () => {
+      beforeEach(async () => {
+        await passwordField.focus();
+        await passwordField.fillIn('test');
+        await passwordField.blur();
+      });
+
+      // THIS test has an unknown intermittent failure in CI
```

Comment:
Thanks for the comment explaining why this is skipped! I'm not super worried about this as these kinds of checks can be removed in the very near future as all authentication-related features are outsourced to Keycloak.

Reply:
Part of the reason I just decided to hang it up on this one.. The other part was *-form library.